src/test/java/org/folio/edge/patron/utils/PatronMockOkapi.java

```diff
@@ -184,6 +193,21 @@ public void getAccountHandler(RoutingContext ctx) {
         .setStatusCode(404)
         .putHeader(HttpHeaders.CONTENT_TYPE, TEXT_PLAIN)
         .end(patronId + " not found");
+    } else if("-1".equals(offset)) {
```

Comment:
I believe our code standards suggest we should have a space before the `if` and opening bracket.

```suggestion
    } else if ("-1".equals(offset)) {
```